core/core.js

```diff
@@ -65,18 +65,15 @@ function createMu (options) {
 
   function dispatch (message, cb) {
     var hash
-    var digest
 
     assert(message, 'dispatch requires a valid message')
     assert(cb && (typeof cb === 'function'), 'dispatch requries a valid callback handler')
     logger.debug('dispatching message: ' + message)
 
     if (!(message.pattern || message.response)) {
-      hash = crypto.createHash('sha256')
-      hash.update('' + cb)
-      digest = hash.digest('hex')
-      router.addRoute(null, {muid: digest, tf: cb, type: 'callback'})
-      router.route({pattern: message, protocol: {path: [digest], trace: [digest], ttl: DEFAULT_TTL}}, cb)
+      hash = md5('' + cb)
```

Comment:
this is highly unneded. I do not think we can generate good digests because of this.
Given this is used to track down each individual call, we might even use https://github.com/mcollina/hyperid.